.travis.yml

```diff
@@ -535,13 +535,15 @@ before_script:
     esac
 # CMocka installation
   - cd ..; mkdir cmocka; cd cmocka; PTH=$(pwd); cd ..;
-  - wget --no-check-certificate https://cmocka.org/files/1.1/cmocka-1.1.0.tar.xz
-  - tar xf cmocka-1.1.0.tar.xz
+  - wget --no-check-certificate https://github.com/argorain/udffsck-test-samples/archive/v1.0-beta.4.tar.gz
+  - tar xzf v1.0-beta.4.tar.gz
+  - cd udffsck-test-samples-1.0-beta.4/cmocka  
+  - tar zxf cmocka-1.1.0-static-patch.tar.gz
   - cd cmocka-1.1.0
   - mkdir build
   - cd build
   - case "$CC" in
-      "tcc") cmake -DCMAKE_INSTALL_PREFIX=$PTH -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=$(which gcc) ../ ;;
+      "tcc") cmake -DCMAKE_INSTALL_PREFIX=$PTH -DCMAKE_BUILD_TYPE=Release -DWITH_STATIC_LIB=ON ../ ;;
```

Comment:
You forgot to enable this `-DWITH_STATIC_LIB=ON` flag for other configurations which have `--static` flag in LDFLAGS. And so it failed on arm and ppc.